main.tf

```diff
@@ -182,7 +190,7 @@ data "aws_iam_policy_document" "ecs_service" {
 resource "aws_iam_role" "ecs_service" {
   count                = local.enable_ecs_service_role && var.service_role_arn == null ? 1 : 0
   name                 = module.service_label.id
-  assume_role_policy   = join("", data.aws_iam_policy_document.ecs_service.*.json)
+  assume_role_policy   = join("", data.aws_iam_policy_document.ecs_service[*].json)
```

Comment:
see above comments and adjust this everywhere

Reply:
as long as we won't run into issues I would prefer to keep it like in the source